web/static_content/html/include/head.html

```diff
@@ -7,16 +7,19 @@
   name="description"
   content="K Semantics of the Ethereum Virtual Machine (EVM)"
 />
-<meta name="keywords" content="runtime, verification, rv, k" />
+<meta
+  name="keywords"
+  content="runtime, verification, rv, k, ethereum virtual machine, evm"
+/>
 <meta name="author" content="EVM Semantics | Runtime Verification Inc" />
 <meta name="robots" content="index, follow" />
 
 <!--favicon icon-->
 <link rel="icon" type="image/png" href="{{$ROOT}}/assets/img/favicon.ico" />
 
-<title>KEVM: Semantics of EVM in K | evm-semantics</title>
+<title>EVM Semantics | Runtime Verification Inc</title>
```

Comment:
Why is this changing? Is this what is displayed at the top of the page? If so, the title is `KEVM: Semantics of EVM in K`, Runtime Verification doesn't need to be mentioned here.

Reply:
Oh I put `Runtime Verification Inc` there because all of our other websites are using title in that format. I can delete it of course.

Reply:
```suggestion
<title>KEVM: Semantics of EVM in K | Runtime Verification, Inc. </title>
```